packages/network-clients/src/clients/networkClient.ts

```diff
@@ -115,16 +115,15 @@ export class NetworkClient {
     const ownDelegation = await this._gqlClient.getDelegation(address, address);
     const delegator = await this._gqlClient.getDelegator(address);
 
-    if (!ownDelegation || !delegator) return BigNumber.from(0);
+    if (!delegator) return BigNumber.from(0);
 
-    const { amount: ownStake } = ownDelegation;
+    const eraNumber = currentEra.toNumber();
+    const ownStake = ownDelegation?.amount;
     const { totalDelegations } = delegator;
 
-    const sortedOwnStake = parseRawEraValue(ownStake, currentEra.toNumber());
-    const sortedTotalDelegations = parseRawEraValue(totalDelegations, currentEra.toNumber());
-    const sortedDelegating = sortedTotalDelegations.current.sub(sortedOwnStake.current);
-
-    return sortedDelegating;
+    const sortedOwnStake = ownStake ? parseRawEraValue(ownStake, eraNumber).after : BigNumber.from(0);
```

Comment:
Should we use the `current` value?

Reply:
Right now the totalDelegations value is only updated when a new delegation is made. So we can either use the after value or set it when a new era occurs.

<img width="1253" alt="Screen Shot 2023-03-09 at 4 50 35 PM" src="https://user-images.githubusercontent.com/8554309/223912112-09a6d78a-e923-47cd-a9ee-d003f5f457da.png">

Reply:
This useDelegating hook is only used in my-account page and in '/delegator/delegating'

Reply:
Make sense